### ISSUE TITLE
This procedure should work for fsproj and vbproj files too

### ISSUE THREAD

**Issue by nosami** [yuv4ik/Mutatio#1]
No reason for it to be limited to csproj!

**Comment by yuv4ik** [comment]
Good idea! In fact both fsproj & vbproj support can be easily implemented.

**Comment by yuv4ik** [comment]
It is actually not that simple as I though.
There is no option to create a `VBNet` project in stable VS for Mac, unless you choose .NET Framework 4.6.x and not Core. Which make it totally useless.
As for `F#` it is not enough just to simply change the .fsproj as it is done for `C#` but I am still investigating it.
